error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/CanonicalConstantFieldNameTest.java

```diff
@@ -0,0 +1,39 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.TestMode;
+import org.junit.jupiter.api.Test;
+
+final class CanonicalConstantFieldNameTest {
+  @Test
+  void replacement() {
+    BugCheckerRefactoringTestHelper.newInstance(CanonicalConstantFieldName.class, getClass())
+        .addInputLines(
+            "A.java",
+            "class A {",
+            "  private static final int number = 1;",
```

Comment:
What if we have a field named `number` and another field already named `NUMBER`? :upside_down_face: